src/matchmaking/book/opening_book.hpp

```diff
@@ -14,7 +14,7 @@ namespace fast_chess {
 class OpeningBook {
    public:
     OpeningBook() = default;
-    explicit OpeningBook(const options::Opening& opening);
+    explicit OpeningBook(const options::Opening& opening, const options::Tournament& tournament);
```

Comment:
passing opening is useless cause its on the torunamentr already

Reply:
fixed 👍